src/kesslerav/protocol2k/io.py

```diff
@@ -13,10 +14,11 @@ class Command(IntEnum):
   Enumerates the supported Protocol 2000 commands
   """
   SWITCH_VIDEO = 1
+  SWITCH_AUDIO = 2
   RECALL_VIDEO_STATUS = 4
   ERROR = 16
   PANEL_LOCK = 30
-  QUERY_OUTPUT_STATUS = 5
+  QUERY_OUTPUT_STATUS = 6 # Query audio status so we can transparently cope with audio only matrices - for video matrices the audio routing should always match the video routing
```

Comment:
I don't believe that this is correct.

Some video matrix switches support switching audio and video from two different inputs to one output. This likely needs to be `QUERY_AUDIO_OUTPUT_STATUS` and the above changed to `QUERY_VIDEO_OUTPUT_STATUS`.

Reply:
Hmm. Damn. So much for that cunning plan. I'm not sure how best to identify audio only matrices then, as mine replies to, say, IDENTIFY MACHINE[1,0] (Request 1st four digits of video machine) with the first four digits of the audio machine. 

It may have to be a user override.